### PR TITLE
New invoice content in AP/AR tests

### PR DESCRIPTION
**Pull request by ylavoie**
Make sure we get the desired customer or vendor before declaring the page loaded to remove some test brittleness.

### CODE REVIEW

File: xt/lib/PageObject/App/AP/Invoice.pm [hunk 1]
```diff
@@ -41,6 +41,7 @@ sub select_vendor {
     $elem->send_keys($vendor);
 
     $self->update;
+    $self->session->page->body->maindiv->wait_for_content(replaces => $elem);
```
Comment (ehuelsmann) [inline xt/lib/PageObject/App/AP/Invoice.pm:44]: Nice find!

(However, I think it makes more sense to do this here: https://github.com/ledgersmb/LedgerSMB/blob/4d02bed8491eb6b4244a09b6244541a7e65d360c/xt/lib/PageObject/App/Invoice.pm#L27)

**Review by ehuelsmann — CHANGES_REQUESTED** [review]

File: xt/lib/PageObject/App/Invoice.pm [hunk 2]
```diff
@@ -21,10 +21,10 @@ sub _verify {
 }
 
 sub update {
-    my ($self) = @_;
+    my ($self,$elem) = @_;
 
     $self->find("*button", text => "Update")->click;
-    $self->session->page->body->maindiv->wait_for_content;
+    $self->session->page->body->maindiv->wait_for_content(replaces => $elem);
```
Comment (ehuelsmann) [inline xt/lib/PageObject/App/Invoice.pm:27]: I understand what you're doing here, but the "Update" button could serve the same purpose, no?

Comment (ylavoie) [reply xt/lib/PageObject/App/Invoice.pm:27]: Yes, if the whole page changes. What if only partial updates are done? Only the Customer/Vendor field seems to be updated.

Comment (ylavoie) [reply xt/lib/PageObject/App/Invoice.pm]: Indeed. Simplified

**Comment by ylavoie** [comment]
Please squash on merge.

**Pull request merged**